src/olympia/accounts/views.py

```diff
@@ -271,3 +274,58 @@ def retrieve(self, request, *args, **kwargs):
             # address has an account associated with it.
             user = STUB_FXA_USER
         return Response(self.get_serializer(user).data)
+
+
+class AccountSuperCreate(JWTProtectedView):
+
+    @waffle_switch('super-create-accounts')
+    def post(self, request):
+        if not acl.action_allowed_user(request.user,
```

Comment:
cool, I'll take a look. Also, it looks like Marketplace did copy over the `acl`.